docs/api/model_selection/learning_curve.rst

```diff
@@ -23,39 +23,36 @@ Classification
 
 In the following example we show how to visualize the learning curve of a classification model. After loading a ``DataFrame`` and performing categorical encoding, we create a ``StratifiedKFold`` cross-validation strategy to ensure all of our classes in each split are represented with the same proportion. We then fit the visualizer using the ``f1_weighted`` scoring metric as opposed to the default metric, accuracy, to get a better sense of the relationship of precision and recall in our classifier.
 
-.. code:: python
+.. plot::
+    :context: close-figs
+    :alt: Learning Curve on the Game dataset using StratifiedKFold
 
     import numpy as np
 
     from sklearn.naive_bayes import MultinomialNB
     from sklearn.model_selection import StratifiedKFold
+    from sklearn.preprocessing import OneHotEncoder
     from yellowbrick.model_selection import LearningCurve
+    from yellowbrick.datasets import load_game
 
-    # Load a classification data set
-    data = load_data('game')
-
-    # Specify the features of interest and the target
-    target = "outcome"
-    features = [col for col in data.columns if col != target]
+    # Load a classification dataset
+    X, y = load_game()
 
     # Encode the categorical data with one-hot encoding
-    X = pd.get_dummies(data[features])
-    y = data[target]
+    X = OneHotEncoder().fit_transform(X)
 
     # Create the learning curve visualizer
-    cv = StratifiedKFold(12)
+    cv = StratifiedKFold(n_splits=12, random_state=42)
```

Comment:
I am unsure if random_state is necessary here... @Kautumn06 your input is needed

Reply:
`random_state=42` has been removed. 

The plot renders identically without this `random_state`.